upgrade-ops-manager/vsphere/pipeline.yml

```diff
@@ -103,7 +103,7 @@ jobs:
       NTP: {{ntp}}
       OPSMAN_NETWORK: {{opsman_network}}
       OPSMAN_VM_FOLDER: {{opsman_vm_folder}}
-      OPSMAN_ADMIN_PASSWORD: {{opsman_admin_password}}
+      OPSMAN_ADMIN_PASSWORD: {{opsman_ssh_password}}
```

Comment:
The key should also be changed from `OPSMAN_ADMIN_PASSWORD`, since that's misleading. Perhaps `OPSMAN_SSH_PASSWORD`, to match the param.